wdio.conf.ts

```diff
@@ -62,8 +62,8 @@ export const config: WebdriverIO.Config = {
       ? {
           platformName: 'Android',
           'appium:automationName': 'UiAutomator2',
-          'appium:deviceName': 'Pixel_6',
-          'appium:platformVersion': '13.0',
+          'appium:deviceName': process.env.DEVICE_NAME || 'Pixel_6',
+          'appium:platformVersion': process.env.PLATFORM_VERSION || '15',
```

Comment:
this is unrelated change to the android Feat/nav3, why introducing this? It breaks current CI flow.